Add Lucky Tip feature

fourohfour opened the pull request:
If you like this feature, you might want to add it :)

fourohfour committed:
Add Lucky Tip feature

ssaamm commented on Randomdoge.py:
```diff
@@ -39,6 +42,10 @@ def pick_random_comment():
             break
 
+def lucky_tip(message, amount):
+    if amount < 100.1:  ## Stop mega payouts
```
Won't this check mean that sometimes people won't get the "Thank you! This will help..." message?

fourohfour commented:
Good point... I would say the best way to rectify would be to have lucky_tip() return a boolean and check for it, if false, then send message.

fourohfour committed:
Fix not getting sent message if check fails

 if check fails in lucky_tip then message would not get sent - fixed.

Thanks to https://github.com/ssaamm
